src/pages/Cart/components/CartCard/CartCard.jsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { HeartIcon } from "../../../../assets/icons";
+
+export function CartCard({ product }) {
+  return (
+    <div className="cart-item">
+      <div className="cart-item-img">
+        <img src={product.img} alt={product.name} loading="lazy" />
```

Comment:
You can de-structure the product details in the function parameter itself and use them.

Reply:
okay will try this